bundles/AdminBundle/Security/User/User.php

```diff
@@ -148,7 +148,7 @@ public function getGoogleAuthenticatorUsername(): string
      *
      * @return string
      */
-    public function getGoogleAuthenticatorSecret(): string
+    public function getGoogleAuthenticatorSecret(): ?string
```

Comment:
As far as I can see, this method never returns `null`, so why this change?

Reply:
Signature changed in the interface https://github.com/scheb/2fa/blob/5.x/src/google-authenticator/Model/Google/TwoFactorInterface.php#L23. Returning null now instead of empty string https://github.com/pimcore/pimcore/commit/69219ce19887a8f77746c59b7eaca8038ac7056f